installer/dev/ProjectReunionInstall.rc

```diff
@@ -17,6 +17,26 @@ PR_FRAMEWORK_X64_RCENTRY
 PR_FRAMEWORK_ARM64_RCENTRY
 #endif
 
+#if defined(PR_MAIN_X86_RCENTRY)
```

Comment:
PR's a bit cryptic (too many possibilities).

SUGGESTION: `PROJECTREUNION_...`

Besides when(!) the coming name change arrives that'll be easy to track down all the places to change

Reply:
I ran into character limits on the macro expression for the resource compiler. I concur that spelling out the name would be better but I'm adjusting to limitations in the resource compiler and feel this is the best option given those limitations.